internal/controller/assets/helper/linperf.sh

```diff
@@ -62,7 +63,7 @@ DISABLE_COLLECTING_HOSTNAME=0 # Default=0 to collect hostname info
 #   that may not be useful towards resolving the issue.  This becomes a problem
 #   when the process of collecting data obscures the real issue.
 ###############################################################################
-SCRIPT_VERSION=2025.04.14
+SCRIPT_VERSION=2025.09.xx
```

Comment:
Suggest to specify the version `2025.09.19`, as in production repo (https://public.dhe.ibm.com/software/websphere/appserv/support/tools/wassdk_scripts/linux/)